rclcpp/test/rclcpp/test_duration.cpp

```diff
@@ -79,6 +79,37 @@ TEST_F(TestDuration, operators) {
   EXPECT_TRUE(time == assignment_op_duration);
 }
 
+TEST_F(TestDuration, operators_with_message_stamp) {
+  builtin_interfaces::msg::Time time_msg;  // 0.1s
+  time_msg.sec = 0;
+  time_msg.nanosec = 100000000;
```

Comment:
can be shorter and more clear?

```suggestion
  rclcpp::Time test_time = rclcpp::Time(0, 100000000u);
  builtin_interfaces::msg::Time time_msg = test_time;
```

Reply:
How about
```suggestion
  builtin_interfaces::msg::Time time_msg = rclcpp::Time(0, 100000000u);  // 0.1s
```